doc/nrf/ug_bootloader.rst

```diff
@@ -102,15 +102,15 @@ Complete the following steps to add a secure bootloader chain to your applicatio
 
    a. Select **Project** > **Configure nRF Connect SDK project**.
    #. Go to **Nordic nRF Connect** and select **Secure Boot** to enable :option:`CONFIG_SECURE_BOOT`.
-   #. Under **Private key PEM file** (:option:`CONFIG_SB_SIGNING_KEY_FILE`), enter the path to the private key that you created.
+   #. Under **Private key PEM file** (:option:`CONFIG_IB_SIGNING_KEY_FILE`), enter the path to the private key that you created.
```

Comment:
We should probably rename ug_bootloader.rst as well?

Reply:
ug_immutable_bootloader.rst?